test/system/add_patient_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'application_system_test_case'
+
+class AddPatientTest < ApplicationSystemTestCase
+  before do
+    @user = create :user, role: 'admin', name: 'Billy'
+    @user2 = create :user, role: 'cm', name: 'Susie'
+
+    @patient = create :patient, name: 'tester',
```

Comment:
So events are created as a byproduct of other actions automatically, and I believe we're testing that behavior in the controller/model tests. I think we can slim this test down a little bit by doing the following after this patient create:

```
@patient.calls.create status: 'Left voicemail'
```

which should create a call via an unrelated stub user, and thenyou can ditch the part in L19-25 where you create a call, log out, and log back in. Might shave off some time on the test, and hone in on the specific behavior we want to test. thoughts?

Reply:
I tried a few variations but the activity log didn't update after using that snippet. But that would be awesome to cut down on this chromedriver stuff.